providers/asrockrack/inventory_test.go

```diff
@@ -0,0 +1,27 @@
+package asrockrack
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_GetInventory(t *testing.T) {
+	device, err := aClient.Inventory(context.TODO())
```

Comment:
is this relying on a package global?

Reply:
This was to have all the handlers defined and registered in the same place https://github.com/bmc-toolbox/bmclib/blob/inventory-firmware/providers/asrockrack/mock_test.go#L75

I'm open to suggestions on how this could be done differently.

Reply:
ah, for a test package, its probably OK